atc/exec/put_inputs.go

```diff
@@ -28,13 +28,13 @@ func NewAllInputs() PutInputs {
 	return &allInputs{}
 }
 
-func (i allInputs) FindAll(artifacts *artifact.Repository) ([]worker.InputSource, error) {
-	inputs := []worker.InputSource{}
+func (i allInputs) FindAll(artifacts *build.Repository) ([]worker.FooBarInput, error) {
```

Comment:
does this need a caller? doesn't look like `i` is used anywhere within the function